All_home_works/hw8/task1.py

```diff
@@ -50,15 +50,17 @@ def _get_data_from_file(self):
         return data
 
     def __getitem__(self, item):
-        return self._get_value_if_exists(item)
+        value = self.data.get(item, None)
+        if value is None:
+            raise KeyError(
```

Comment:
You don't need to raise it yourself, you can just return self.data[item]

Reply:
But I wanted to have a more beautiful description of this error. Than:
 `value = self.data[item]`
` KeyError: 'not_existing'`